frontend/www/js/omegaup/contest/edit.js

```diff
@@ -169,24 +169,40 @@ OmegaUp.on('ready', function() {
               },
               'add-user': function(ev) {
                 let contestants = [];
+                let contestantsWithError = [];
+                let usersAdded = false;
                 if (ev.contestants !== '')
                   contestants = ev.contestants.split(',');
                 if (ev.contestant !== '') contestants.push(ev.contestant);
-                Promise.all(
+                Promise.allSettled(
                   contestants.map(contestant =>
                     API.Contest.addUser({
                       contest_alias: contestAlias,
                       usernameOrEmail: contestant.trim(),
                     }),
                   ),
                 )
-                  .then(function() {
-                    UI.success(T.bulkUserAddSuccess);
-                    refresh(ev, API.Contest.users, 'users');
+                  .then(results => {
+                    results.forEach(result => {
+                      refresh(ev, API.Contest.users, 'users');
+                      if (result.status === 'rejected') {
+                        contestantsWithError.push(result.reason.userEmail);
+                      }
+                      if (result.status === 'fulfilled' && !usersAdded) {
+                        usersAdded = true;
+                      }
```

Comment:
```suggestion
                      usersAdded |= (result.status === 'fulfilled');
```

alternativamente, puedes saber si se agregaron usuarios si `contestantsWithError.length != contestants.length`, así que ese booleano se puede eliminar por completo!

y ahora que veo el resto del código, dónde se está usando esta variable?

Reply:
Creo que sólo lo pensé. Quería mandar dos textos distintos:
- Uno si se había agregado al menos un usuario correctamente.
- Otro si todos se habían ido a error.

Pero veo que no es necesario. Así que mejor quito esta lógica.